client/src/main/scala/drt/client/SPAMain.scala

```diff
@@ -78,7 +86,9 @@ object SPAMain extends js.JSApp {
         dynRenderR((page: TerminalsDashboardLoc, router) => {
           TerminalsDashboardPage(None, router, page)
         })
-      val terminal: dsl.Rule = dynamicRouteCT(("#terminal" / string("[a-zA-Z0-9]+") / string("[a-zA-Z0-9]+") / string("[a-zA-Z0-9]+") / string(".+").option).caseClass[TerminalPageTabLoc]) ~>
+      val terminal: dsl.Rule = dynamicRouteCT(
+        ("#terminal" / string("[a-zA-Z0-9]+") / string("[a-zA-Z0-9]+") / string("[a-zA-Z0-9]+") / string(".+").option
```

Comment:
would it be possible to extract these regexs to vals to give them names or are they too generic?